package.json

```diff
@@ -6,11 +6,11 @@
   "scripts": {
     "pretest": "eslint .",
     "test": "nyc --report-dir ./artifacts/coverage --reporter=lcov mocha --reporter mocha-multi-reporters --reporter-options configFile=./mocha.config.json --recursive --timeout 4000 --retries 1 --exit --allow-uncaught true --color true",
-    "semantic-release": "semantic-release pre && npm publish && semantic-release post"
+    "semantic-release": "./node_modules/.bin/semantic-release"
   },
   "repository": {
     "type": "git",
-    "url": "git@github.com:screwdriver-cd/artifact-bookend.git"
+    "url": "git+https://github.com:screwdriver-cd/artifact-bookend.git"
```

Comment:
Is the format of url  right? other packages like `git+https://github.com/screwdriver-cd/`
https://github.com/screwdriver-cd/store/blob/master/package.json#L15
https://github.com/screwdriver-cd/models/blob/master/package.json#L12
https://github.com/screwdriver-cd/artifacts-unzip-worker/blob/main/package.json#L14

Reply:
thx.

Reply:
I fixed it.